scripts/generate_conditions.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+_condition_strings = [
+    "ArnEquals",
+    "ArnNotEquals",
+    "ArnLike",
+    "ArnNotLike",
+    "Bool",
+    "DateEquals",
+    "DateNotEquals",
+    "DateLessThan",
+    "DateLessThanEquals",
+    "DateGreaterThan",
+    "DateGreaterThanEquals",
+    "IpAddress",
+    "NotIpAddress",
+    "Null",
+    "NumericEquals",
+    "NumericNotEquals",
+    "NumericLessThan",
+    "NumericLessThanEquals",
+    "NumericGreaterThan",
+    "NumericGreaterThanEquals",
+    "StringEquals",
+    "StringNotEquals",
+    "StringEqualsIgnoreCase",
+    "StringNotEqualsIgnoreCase",
+    "StringLike",
+    "StringNotLike",
+]
+
+_condition_qualifier_strings = ["ForAnyValue", "ForAllValues"]
+
+
+def make_condition(type_name: str, condition_name: str) -> None:
+    print(f"class {type_name}(ConditionElement):")
+    print(f"    condition = '{condition_name}'")
+    print("")
+    print("")
+    print(f"class {type_name}IfExists(ConditionElement):")
+    print(f"    condition = '{condition_name}IfExists'")
+    print("")
+    print("")
+
+    # Previous dynamic creation in aws.py
+    #
+    # globals()[type_name] = type(
+    #    type_name, (ConditionElement,), dict(condition=condition_name)
+    # )
+    # globals()[type_name + "IfExists"] = type(
+    #    type_name + "IfExists",
+    #    (ConditionElement,),
+    #    dict(condition=condition_name + "IfExists"),
+    # )
+
+
+# Create condition classes
+for i in _condition_strings:
```

Comment:
`i` isn't a good name for a variable.  But since it's only in `scripts/` and existed for a long time :shrug: